languagetool-http-client/src/test/java/org/languagetool/remote/RemoteResultTest.java

```diff
@@ -0,0 +1,28 @@
+package org.languagetool.remote;
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+public class RemoteResultTest {
+
+  @Test
+  public void getLanguageDetectedCodeOutputNotNull() {
+    final RemoteServer remoteServer = new RemoteServer("LanguageTool", "", "");
+    final List matches = new ArrayList();
+    final RemoteResult objectUnderTest = new RemoteResult("English", "en", "en", "English", matches, remoteServer);
+    final String retval = objectUnderTest.getLanguageDetectedCode();
+    Assert.assertEquals(retval, "en");
+  }
+
+  @Test
+  public void getLanguageDetectedNameOutputNotNull() {
+    final RemoteServer remoteServer = new RemoteServer("LanguageTool", "", "");
+    final List matches = new ArrayList();
+    final RemoteResult objectUnderTest = new RemoteResult("English", "en", "en", "English", matches, remoteServer);
+    final String retval = objectUnderTest.getLanguageDetectedName();
+    Assert.assertEquals(retval, "English");
```

Comment:
Could we combine these tests in to one? They both have the same arrangement of objects prior to the Retval. 

`  @Test
  public void getLanguageDetectedCodeOutputNotNull() {
    final RemoteServer remoteServer = new RemoteServer("LanguageTool", "", "");
    final List matches = new ArrayList();
    final RemoteResult objectUnderTest = new RemoteResult("English", "en", "en", "English", matches, remoteServer);
    Assert.assertEquals(objectUnderTest.getLanguageDetectedCode(), "en");
    Assert.assertEquals(objectUnderTest.getLanguageDetectedName(), "English");
  }`

Reply:
Yes! Done.